src/components/Events.module.scss

```diff
@@ -15,8 +17,22 @@
   width: 100%;
   gap: 30px;
   justify-content: space-between;
+  max-width: 100%;
 }
 
 .sport {
   font-weight: bold;
 }
+
+@media only screen and (max-width: $screen-md-min) {
+  .event {
+    margin-left: auto;
+    margin-right: auto;
+    height: 200px;
+    width: 343px;
+    border: 2px solid #333;
+    padding-top: 5px;
+    padding-left: 10px;
+    border-radius: 10px;
+  }
+}
```

Comment:
1. Drop the screen; it is not necessary and does look like chatgpt generated
2. Always prefer `min-width` which looks like you are doing the development "mobile-first". Mobile first is an idea that you start with web development on phone size and then gradually widen it and add more necessary CSS. No one does that in reality, but `min-width` is the only thing that sticks.
 - whenever you will open some project, make sure what they use and stick to it. NEVER, mix min and max together. Only Kosek does that